static/styles.css

```diff
@@ -30,6 +30,30 @@ main {
   width: 240px;
```

Comment:
Adding `white-space: nowrap` on the `<nav class="paginate-container">` should do the trick to prevent wrapping

Reply:
This solution is not enough for low res screens, because Previous/Next buttons are not in visible scope.

Reply:
Idea : use media queries to display properly the pagination buttons. WIP

Reply:
commit b16551056ee1c6b117b5b3fe653a71b082cb9ef6